frog-utils/src/dashboards/progress.js

```diff
@@ -50,24 +50,27 @@ export const LineChart = ({
 );
 
 const TIMEWINDOW = 5;
+let timingData;
```

Comment:
- This is used inside the Viewer component, but defined outside. Not a good practice. 

- If the Viewer component is called for the first time after the activity has ended (for example if the page is refreshed) timingData will be undefined.